src/dataServices/dynamoDbBundleService.ts

```diff
@@ -270,6 +286,81 @@ export class DynamoDbBundleService implements Bundle {
         }
     }
 
+    private async updatedReferences(requests: BatchReadWriteRequest[], lockedItems: ItemRequest[]) {
+        const idToVersionId: Record<string, string> = {};
+        lockedItems.forEach((itemRequest: ItemRequest) => {
+            if (itemRequest.operation === 'update' && itemRequest.vid) {
```

Comment:
Is this `vid` coming from the bundle request itself or is it pull from dynamo in the `lockItems` call?

Reply:
The `vid` is pulled from dynamo in the `lockItems` call